### DISCUSSION TITLE
Point Cloud Extraction

### DISCUSSION THREAD

**Discussion by hootan-a** [ApolloAuto/apollo#15076] [Q&A]
Dear Apollo Team
this is my system information
    Linux Mint 21:
    Apollo installed with docker:
    Apollo version 8.0:

When I run this command to generate pcd files I get "Segmentation fault" error, I have no idea why this happens?
this is my output of cyber_record_parser:

cyber_record_parser \--bag_file demo_3.5.record \--out_folder data/bag/output \--cloud_topic /apollo/sensor/lidar128/compensator/pointcloud2:

![image](https://github.com/ApolloAuto/apollo/assets/141530653/e3f586b6-4d85-4879-acd4-2f296278b6b2)

**Comment by daohu527** [comment]
Duplicated with issue, I think this question is more suitable in issue, closed soon!
